Log message:
irqreturn_t is defined in <linux/interrupt.h>

Modified files:
    linux/include/asm-m68k:
	atari_stdma.h 


Index: linux/include/asm-m68k/atari_stdma.h
Stats: 2 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/include/asm%2dm68k/atari_stdma%2eh.diff?r1=1.2&r2=1.3
========================================================================
--- linux/include/asm-m68k/atari_stdma.h	9 May 2003 08:39:20 -0000	1.2
+++ linux/include/asm-m68k/atari_stdma.h	9 May 2006 18:47:28 -0000	1.3
@@ -3,7 +3,7 @@
 #define _atari_stdma_h
 
 
-#include <asm/irq.h>
+#include <linux/interrupt.h>
 
 
 /***************************** Prototypes *****************************/
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

